### ISSUE TITLE
Who are Qubes' users: Past, Present, and (targeted)Future?

### ISSUE THREAD

**Issue by ninavizz** [QubesOS/qubes-issues#4758]
Creating this issue as a new discussion, focused entirely around strategic goals of defining who users are/were/willbe. This issue should never be tied to a release, nor to any aspects of building the thing that is Qubes. It will also never be cited directly in documentation, so should not be set-up as a dependency for anything related to the website, readmes, etc.

This issue picks-up where #1906 began, diverged, spun in some elegant circles, then re-focused on project deliverable needs. Defining who users are should focus on business/social goals: who is the product seeking to serve, and why?

Acceptance criteria: A set of personas created, agreed to, and categorized as "Past" "Present" and "Targeted (ideally within a timeframe)" users—with reasons cited for adoption, attrition, or anticipated value for future users. Nothing has to be fancy, just done/defined. Later, these artifacts can and should all be referenced in planning, design, prioritization, and other such strategic activities.

Can/will write more, later—but just wanted to get this Issue created as a first step independent of any PR or other technical deliverable.

@marmarek I'd recommend categorizing this as "Business," or creating a new category for "Strategy."

**Comment by mfc** [comment]
hey nina, this may overlap a bit with [this existing ticket](https://github.com/QubesOS/qubes-issues/issues/1906) and in particular [my stalled effort there](https://github.com/QubesOS/qubes-issues/issues/1906#issuecomment-217473679). maybe we can steal from there and use your much stronger framing.

**Comment by andrewdavidwong** [comment]
A couple of minor changes:
- Retitled the issue to make it clear what's actionable here.
- Looks more like a `project management` issue than a `business` issue, but I'm open to being convinced otherwise.

Looking forward to seeing the first draft!

**Comment by ninavizz** [comment]
@mfc Yep, I cited the Issue you describe, in my initial description. Because that Issue is closed, and I wanted to more clearly frame the objective, I wanted to create this as a "New" revival of the question. 

I absolutely intend to heavily borrow from what's already (and awesomely!) been done in 1906. :)

**Comment by andrewdavidwong** [comment]
> Because that Issue is closed

#1906 is actually still open, FWIW.

**Comment by ninavizz** [comment]
True—but it was created to inform documentation. The objective of this Issue is to define business and product interests, for prioritization.
